installer/dbscript/dbscript-2.sql

```diff
@@ -4256,6 +4256,7 @@ VALUES ('apiv2_time_time_sheet_config', 'API-v2 Time - Time Sheet Start Day Conf
        ('apiv2_time_my_timesheet', 'API-v2 Time - My Timesheet', 1, 1, 1, 0),
        ('apiv2_time_timesheet_action_log', 'API-v2 Time - Timesheet action log', 1, 0, 0, 0),
        ('apiv2_time_timesheet_comment', 'API-v2 Time - Timesheet comment', 1, 1, 1, 0),
+       ('apiv2_time_time_config_format', 'API-v2 Time - Time Config Format', 1, 0, 0, 0),
```

Comment:
```suggestion
       ('apiv2_time_config_time_format', 'API-v2 Time - Time Config Format', 1, 0, 0, 0),
```
refactor according to this